.eslintrc.js

```diff
@@ -70,6 +70,7 @@ module.exports = {
    */
   rules: {
     ...baseline.rules,
+    'prefer-object-spread': 'off',
```

Comment:
`_objectSpread2` is at the top of stack traces in performance profiles and this rule keeps re-writing my `Object.assign` into `{ ... }`.

Reply:
I just realized that I've been doing benchmarks on our docs site, and our docs site uses a different babel config than our production release builds, which means our production builds have the `_objectSpread2` optimized away, but not our docs. So our components in the docs run slower than they will when used by our library users.

Reply:
Can we optimize `_objectSpread2` in both?

Reply:
Yes, but I'm waiting for core to merge https://github.com/mui/material-ui/pull/37461 so I can mirror their config.